.circleci/config.yml

```diff
@@ -42,3 +62,6 @@ workflows:
           filters:
             branches:
               only: production
+      - artifacts:
```

Comment:
Do we want this job to fall under the `build_and_deploy` category? Wouldn't this happen automatically at every commit that is pushed up to a remote branch in this repo?